usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -539,6 +560,57 @@
         global_temp.ref_country_code AS rcc
             ON (rcc.country_code = UPPER(bs.sub_legal_entity_country_code)
                 AND bs.sub_legal_entity_country_code IS NOT NULL)
+    LEFT OUTER JOIN
+        global_temp.cd_state_grouped pop_cd_state_grouped ON (
+            pop_cd_state_grouped.state_abbreviation=UPPER(bs.sub_place_of_perform_state_code)
+            AND pop_cd_state_grouped.congressional_district_no <> '90'
```

Comment:
Same with this random 90. Might be helpful for future maintainers to store this in a variable with a descriptive name or leave a comment. Additionally, you may want to go the variable route because 90 is repeated multiple times here.

Reply:
I can include a comment to explain the `90` but don't want to go down the variable route as explained in the other comment.

Reply:
ok